CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -129,56 +129,70 @@ public PluginParameters createParameters() {
      * file imports.
      *
      * @param title Title to add to status dialog
-     * @param importedRows Number of rows successfully imported (from valid
-     * files)
-     * @param validFilenames List of filenames that were imported from
-     * @param invalidFilenames List of files that couldn't be opened/parsed. We
-     * try to limit this possibility by pre-screening files during the initial
-     * file selection.
+     * @param importedRows Number of rows successfully imported (from validFilenames)
+     * @param validFilenames List of valid filenames that were imported from
+     * @param emptyFilenames List of filenames that were requested to import that contained no data rows
+     * @param invalidFilenames List of files that couldn't be opened/parsed. We try to limit this possibility by
+     * pre-screening files during the initial file selection
+     * @param emptyRunConfigs List of run config names that were found to not have vertex of transaction mappings
+     * defined
      */
-    private void displaySummaryAlert(final int importedRows, final List<String> validFilenames, final List<String> invalidFilenames) {
+    private void displaySummaryAlert(final int importedObjects, final int importedRows, final List<String> validFilenames,
+                                     final List<String> emptyFilenames, final List<String> invalidFilenames,
+                                     final List<String> emptyRunConfigs) {
         Platform.runLater(() -> {
             final StringBuilder sbHeader = new StringBuilder();
             final StringBuilder sbMessage = new StringBuilder();
 
-            final String fileFiles = (validFilenames.size() == 1) ? "file" : "files";
-
-            if (importedRows > 0) {
-                // At least 1 row was successfully imported. List all successful file imports, as well as any files that there were
-                // issues for. If there were any files with issues use a warning dialog.
-                sbHeader.append(String.format("Imported %d rows of data from %d " + fileFiles, importedRows, validFilenames.size()));
-                sbMessage.append("The following " + fileFiles + " contained data: ");
-
+            if (importedObjects > 0) {
+                // At least 1 object was successfully imported. List all successful file imports, as well as any files
+                // that there were issues for. If there were any files with issues use a warning dialog.
+                sbHeader.append(String.format("Extracted data from %d rows in %d files", importedRows, validFilenames.size()));
```

Comment:
For one file it says "files" again. This was corrected previously and looks like the code block above had the solution. Can this be added back in @serpens24?

Reply:
Added code to determine words to use